### DISCUSSION TITLE
Integration

### DISCUSSION THREAD

**Discussion by gunner4361** [Yamato-Security/hayabusa#1184] [Ideas]
Any thoughts on integration with other IP reps

https://www.abuseipdb.com/

https://www.ipqualityscore.com/ip-reputation-check

**Comment by YamatoSecurity** [comment]
@gunner4361 Thank you for the suggestion. I will look into adding it to Takajo as we are gong to focus Hayabusa on the scanning part and Takajo for the data enrichment, IP lookups, etc... The only exception now is geoIP lookups which we do in Hayabusa because it is an offline database so does not slow down scanning like looking up things through an API would.
